apps/android/LiveVideoApp/app/src/main/java/com/twilio/livevideo/app/view/SignInPasscodeFragment.kt

```diff
@@ -0,0 +1,36 @@
+package com.twilio.livevideo.app.view
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.twilio.livevideo.app.databinding.FragmentSignInPasscodeBinding
+import dagger.hilt.android.AndroidEntryPoint
+import timber.log.Timber
+
+@AndroidEntryPoint
+class SignInPasscodeFragment : SignInBaseFragment() {
+
+    lateinit var viewDataBinding: FragmentSignInPasscodeBinding
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        viewDataBinding = FragmentSignInPasscodeBinding.inflate(inflater, container, false)
+        viewDataBinding.viewModel = viewModel
+        viewDataBinding.lifecycleOwner = this
+        return viewDataBinding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        Timber.d("SignInPasscodeFragment onViewCreated")
```

Comment:
Ohhh I just put this log here to fix an unexpected behavior when I was implementing the navigation system. But I forgot to remove it. Even tho, the logs are only available on Debug builds.

Reply:
Ok. Yeah since this is reference code I think it will be a little simpler to read the code without the logging. 

So I would recommend removing all logging that isn't useful for the customer.

I could see including logging for something like a special condition or error handling that we think will be helpful to customers. But I would remove logging that was only added to debug a specific problem.

Reply:
Done!